src/test/java/AuthorisationTest.java

```diff
@@ -1,9 +1,36 @@
 import org.junit.jupiter.api.Test;
 import poort8.ishare.core.Authorisation;
-
 public class AuthorisationTest {
+
+    @Test void AuthorisationProcess() {
+        String accessToken = Authorisation.GetAccessToken("EU.EORI.NL888888881");
+
+        if (accessToken != null) {
+            System.out.println("Access token: " + accessToken);
+        } else {
+            System.out.println("AuthorisationProcess test failed: could not get access token");
+            return;
+        }
+
+        String delegationEvidence = Authorisation.GetDelegationEvidence(accessToken, "poort8.iSHARE.service", "poort8.iSHARE.service", "1", "read");
+        if (delegationEvidence != null && !delegationEvidence.isEmpty()) {
+            System.out.println("Delegation evidence: " + delegationEvidence);
+        } else {
+            System.out.println("AuthorisationProcess test failed: could not get delegation evidence");
+            return;
+        }
+
+        boolean access = Authorisation.VerifyAccess(delegationEvidence, "EU.EORI.NL888888881", "poort8.iSHARE.service", "poort8.iSHARE.service", "1", "read");
```

Comment:
In a test I would expect some asserts. I don't think this is blocking for this PR, so next time :)